.appveyor.yml

```diff
@@ -30,6 +30,7 @@ cache:
 
 install:
   - '%CYG_ROOT%\setup-%CYG_ARCH%.exe -qnNdO -R %CYG_ROOT% -s http://cygwin.mirror.constant.com -l %CYG_ROOT%/var/cache/setup %PACKAGES%'
+  - "python.exe -m pip install gcovr"
```

Comment:
This is not really necessary, since we can't use coveralls on AppVeyor for now anyway (starting with the limitation in the profiling package, which hard codes "travis" as CI provider right now)